### DISCUSSION TITLE
New contribution guideline help

### DISCUSSION THREAD

**Discussion by jingcai86** [plouc/nivo#2543] [General]
Hi nivo community,

My name is Jing, I am from Cisco network team. We have been using nivo internally for almost a year. And we are trying to contribute back to the open source community.

We have been building our internal chart with some of nivo custom layers based on our internal theme as well as accessibility requirements.

However, some of the feature such has adding event handling for onFocus event for accessibility must be done within nivo source, that's why we want to contribute it back.

To facilitate such process, is there resource on what we can submit on PR, and format how to make PR, or any guideline that can help us to work with you all?

Thanks

Jing

**Comment by plouc** [comment]
Hi @jingcai86,

First, sorry for the late reply, there's no such guide at the moment, the only guide we have is: https://github.com/plouc/nivo/blob/master/CONTRIBUTING.md.

For PRs, explaining the motivation behind the change should be enough, along with some design notes if the implementation is not straightforward.
